transpiler/transpiler_test.go

```diff
@@ -13,6 +13,7 @@ var sketches = []string{
 	"blink",
 	"button",
 	"fade",
+	"language-basics"
```

Comment:
There's a syntax error here, a missing ',' before newline in this composite literal.

It looks like this project doesn't have CI at this time, that's why it's not automatically caught.